app/graphql/types/base_argument.rb

```diff
@@ -2,5 +2,8 @@
 
 module Types
   class BaseArgument < GraphQL::Schema::Argument
+    def initialize(*args, camelize: false, **kwargs, &block)
+      super
+    end
```

Comment:
**question:** argument support

Does this definitely work for arguments? I can see that this is the suggested code for fields, but for arguments I can only see the suggestion for how to do it on a per-argument basis

Reply:
I couldn't find any documentation for arguments, however since we're defining `Types::BaseArgument` as a argument class in our base field, I'm confident we're overriding the argument definition in the same way as we're doing for fields.